packages/components/src/Form/Fields/FieldToggleSwitch/FieldToggleSwitch.test.tsx

```diff
@@ -30,24 +30,41 @@ import { assertSnapshot } from '@looker/components-test-utils'
 import { FieldToggleSwitch } from './FieldToggleSwitch'
 
 test('A FieldToggleSwitch', () => {
+  assertSnapshot(<FieldToggleSwitch id="FieldToggleSwitchID" label="👍" />)
+})
+
+test('A FieldToggleSwitch turned on', () => {
+  assertSnapshot(
+    <FieldToggleSwitch id="FieldToggleSwitchID" label="👍" on={true} />
+  )
+})
+
+test('A FieldToggleSwitch with validationMessage', () => {
   assertSnapshot(
-    <FieldToggleSwitch label="👍" name="thumbsUp" id="thumbs-up" />
+    <FieldToggleSwitch
+      id="FieldToggleSwitchID"
+      label="Toggle Switch"
+      validationMessage={{ message: 'validation Message', type: 'error' }}
+    />
   )
 })
 
-test('A FieldToggleSwitch with label aligned left', () => {
+test('A FieldToggleSwitch turned on', () => {
```

Comment:
Test description is wrong.

```suggestion
test('A FieldToggleSwitch disabled', () => {
```